modules/virt-about-nmstate.adoc

```diff
@@ -27,5 +27,5 @@ Node networking is monitored and updated by the following objects:
 
 [NOTE]
 ====
-If your {product-title} cluster uses OVN-Kubernetes as the network plugin, you cannot attach a Linux bridge or bonding to the default interface of a host because of a change in the host network topology of OVN-Kubernetes. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.
+If your {product-title} cluster uses OVN-Kubernetes as the network plugin, configuration changes to the interface is not allowed. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.
```

Comment:
Small suggestion to write this with more user focus and to use active voice:
```suggestion
If your {product-title} cluster uses OVN-Kubernetes as the network plugin, you cannot make configuration changes to the interface. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.
```

Totally optional and up to you which wording you like better.